include/openthread/openthread.h

```diff
@@ -35,17 +35,6 @@
 #ifndef OPENTHREAD_H_
 #define OPENTHREAD_H_
 
-#include <openthread/crypto.h>
-#include <openthread/dataset.h>
-#include <openthread/instance.h>
-#include <openthread/ip6.h>
-#include <openthread/link.h>
-#include <openthread/message.h>
-#include <openthread/netdata.h>
-#include <openthread/tasklet.h>
-#include <openthread/thread.h>
-#include <openthread/types.h>
-
 #ifdef __cplusplus
 extern "C" {
 #endif
```

Comment:
Remove @defgroup api-types Types

Add @defgroup api-error Error wherever you want it to go